shared/src/metabase/mbql/schema.cljc

```diff
@@ -15,14 +15,16 @@
    [(:require
      [clojure.core :as core]
      [clojure.set :as set]
-     ["moment" :as moment]
-     ["moment-timezone" :as mtz]
      [metabase.mbql.schema.helpers :as helpers :refer [is-clause?]]
      [metabase.mbql.schema.macros :refer [defclause one-of]]
+     [moment :as moment]
```

Comment:
i think you'll need to conditionally require these in cljs branched code. And the correct form should be "moment" not moment.

Reply:
it's currently in `cljs` branch of require.

> the correct form should be "moment" not moment.

I can run `yarn run test-cljs` with just when importing without the double-quote.. Is it a convention to have it in quote?

Reply:
I thought it was required. But perhaps there have been changes since I was last deeply in cljs. But the shadow cljs users guide seems to say that the correct form should be the string name just as in regular js code: https://shadow-cljs.github.io/docs/UsersGuide.html#_using_npm_packages

Reply:
I updated to follow the convention